src/utils/isStandardSyntaxMediaFeature.js

```diff
@@ -16,5 +16,8 @@ export default function (mediaFeature) {
   // SCSS or Less interpolation
   if (hasInterpolation(mediaFeature)) { return false }
 
+  // SCSS variables
+  if (/\$.+?/.test(mediaFeature)) { return false }
```

Comment:
> So: What if we just outlaw strings that contain variables and an operator?

SGTM. I was going to say this too. I was wondering though whether just checking for a operator would suffice (e.g. `*`, `/`, `-` and `+`)?